telemetry/telemetry_record.py

```diff
@@ -1,25 +1,80 @@
-"""Record WebPageReplay
+"""Record WebPageReplay, modify, and record page load times
 
 Records running a set of target urls on the remote tablet, modifies cacheable
-objects, then re-runs to see page load time differences.
+objects, then re-runs to see page load time differences. Writes 1 har file for
+each url and modified url in page_load_time/data/hars/
 
-Usage: sudo python record_wpr.py
+Usage: sudo python telemetry_record.py -t 2
 
 Notes:
     Assumes - telemetry is in CHROMIUM_SRC/tools/telemetry/telemetry
             - page_sets is in CHROMIUM_SRC/tools/perf/page_sets
             - modify_wpr_delays.py is in CHROMIUM_SRC/third_party/webpagereplay
+            - url data set is in PLT_SRC/telemetry/targer_urls, line delimited
 
-TODO: Measure page load time before and after modifying
-TODO: Make sure to only get the most recent wpr file when moving / modifying
-TODO: Clean up path dependencies, move them to the top
+Workflow:
+    1. Clean up old session data
+    2. Read target_urls
+    3. Write a page_set for each target url and its modified version to
+       CHROMIUM_SRC/tools/perf/page_sets
+    4. Creates *.json files to point to archive files for modified urls
```

Comment:
This is just a benchmark/wpr technicality. Chromium uses a little .json file to point to the actual archived site. Because I manually create pc archives, I have to manually make the .json files that point to these modified archive files. They literally contain the path to the wpr file. Let me know if I need to clarify further